testing/tests/DevExpress.ui.widgets.form/form.tests.js

```diff
@@ -4032,6 +4032,25 @@ function getColsCountFromDOM($form) {
     return result;
 }
 
+QUnit.test('Setting "colCountByScreen" property of group with dot notation should update layout', function(assert) {
+    const $form = $('#form').dxForm({
+        screenByWidth: () => 'md',
+        items: [
+            {
+                itemType: 'group',
+                colCountByScreen: { md: 3 },
+                items: [{ dataField: 'name' }, { dataField: 'email' }, { dataField: 'age' }]
```

Comment:
is it necessary got the test? Config should be minified as possible

Reply:
yes its all necessary